### PR TITLE
goldendict: whitelist path to documentation and locales

### PR DESCRIPTION
**Pull request by jose1711**

### CODE REVIEW

File: etc/profile-a-l/goldendict.profile [hunk 1]
```diff
@@ -22,6 +22,9 @@ whitelist ${HOME}/.goldendict
 whitelist ${HOME}/.cache/GoldenDict
 # The default path of dictionaries
 whitelist /usr/share/stardict/dic
+# The default path to documentation and locales
+whitelist /usr/share/goldendict/help
+whitelist /usr/share/goldendict/locale
```
Comment (rusty-snake) [inline etc/profile-a-l/goldendict.profile:27]:
```suggestion
whitelist /usr/share/goldendict
```

I guess everything in `/usr/share/goldendict` is necessary for `goldendict`.

File: etc/profile-a-l/goldendict.profile [hunk 2]
```diff
@@ -22,6 +22,7 @@ whitelist ${HOME}/.goldendict
 whitelist ${HOME}/.cache/GoldenDict
 # The default path of dictionaries
 whitelist /usr/share/stardict/dic
+whitelist /usr/share/goldendict
```
Comment (kmk3) [inline etc/profile-a-l/goldendict.profile:25]:
```suggestion
whitelist /usr/share/goldendict
whitelist /usr/share/stardict/dic
```

Sort

File: etc/profile-a-l/goldendict.profile [hunk 3]
```diff
@@ -21,6 +21,7 @@ mkdir ${HOME}/.cache/GoldenDict
 whitelist ${HOME}/.goldendict
 whitelist ${HOME}/.cache/GoldenDict
 # The default path of dictionaries
+whitelist /usr/share/goldendict
 whitelist /usr/share/stardict/dic
```
Comment (rusty-snake) [inline etc/profile-a-l/goldendict.profile:25]:
```suggestion
whitelist /usr/share/goldendict
# The default path of dictionaries
whitelist /usr/share/stardict/dic
```
The comment was about `/usr/share/stardict/dic`.

**Comment by kmk3** [comment]
Squashed and merged, thanks.

**Pull request merged**